src/library/dtls.hpp

```diff
@@ -69,12 +69,13 @@ static constexpr uint32_t kDtlsHandshakeTimeoutMax = 60;
 
 struct DtlsConfig
 {
-    bool      mEnableDebugLogging = false;
-    int       mLogLevel           = 3;
-    ByteArray mPSK;
-    ByteArray mOwnKey;
-    ByteArray mOwnCert;
-    ByteArray mCaChain;
+    bool        mEnableDebugLogging = false;
+    int         mLogLevel           = 3;
+    ByteArray   mPSK;
+    ByteArray   mOwnKey;
+    ByteArray   mOwnCert;
+    ByteArray   mCaChain;
+    std::string mHostname;
```

Comment:
Ditto: this is not the hostname of this device, but should be the hostname of the peer device
```suggestion
    std::string mPeerHostname;
```